packages/mdc-textfield/icon/test/foundation.test.ts

```diff
@@ -148,7 +148,6 @@ describe('MDCTextFieldIconFoundation', () => {
     const evt = {
       target: {},
       type: 'click',
-      preventDefault: () => {}
```

Comment:
@asyncLiz is this intended?

Reply:
Yes, it needs to be there because the icon foundation calls preventDefault() to ensure that clicks don't bubble up to the host label and focus the input.

We're not actually providing a real event, so we have to mock that function.

Reply:
Okay, I'll add that in internal and resync